requirements/dev-requirements.txt

```diff
@@ -20,6 +20,7 @@ deepdiff
 scikit-learn==1.0.2; python_version >= '3.7'
 # pandas 1.3.5 is the last version to support Python 3.7
 pandas==1.3.5; python_version >= '3.7'
+prince==0.7.1
```

Comment:
Again asking - can we relax this requirement?

Reply:
Yes, I will only need to make some changes in the diversity_score for cat features